components/resources/wallet_strings.grdp

```diff
@@ -84,6 +84,7 @@
   <message name="IDS_BRAVE_WALLET_SWAP_FEES_NOTICE" desc="Swap fees notice">Quote includes a 0.875% Brave fee.</message>
   <message name="IDS_BRAVE_WALLET_BUTTON_CONTINUE" desc="General Continue button">Continue</message>
   <message name="IDS_BRAVE_WALLET_BUTTON_COPY" desc="General Copy button">Copy</message>
+  <message name="IDS_BRAVE_WALLET_BUTTON_COPIED" desc="General Copied button">Copied</message>
```

Comment:
Perhaps a more useful description for translators would be "Confirmation that text has been copied to clipboard"